src/utils/accountService.js

```diff
@@ -0,0 +1,353 @@
+   */
+  async _generateECDSA(async, balance, num, startup) {
+    let ecdsaAccountNumCounter = 1002;
+    const params = [];
+
+    if (!async) this._logAccountTitle();
+
+    for (let i = 0; i < num; i++) {
+      let wallet = hethers.Wallet.createRandom();
+      if (startup && this.privateKeysECDSA[i]) {
+        wallet = new hethers.Wallet(this.privateKeysECDSA[i]);
+      }
+      params.push(await this._createAccount(
+        async,
+        ecdsaAccountNumCounter++,
+        balance,
+        startup,
+        wallet
+      ));
+    }
+
+    if (!async) {
+      this._logAccountivider();
+    }
+    return params;
+  }
+
+  /**
+   * @internal
+   * Generate Alias ECDSA accounts.
+   * @param {boolean} async
+   * @param {number} balance
+   * @param {number} num
+   * @param {boolean} startup
+   */
+  async _generateAliasECDSA(async, balance, num, startup) {
+    let aliasedAccountNumCounter = 1012;
+    const params = [];
+
+    if (!async) this._logAliasAccountTitle();
+
+    for (let i = 0; i < num; i++) {
+      let wallet = ethers.Wallet.createRandom();
+      if (startup && this.privateKeysAliasECDSA[i]) {
+        wallet = new ethers.Wallet(this.privateKeysAliasECDSA[i]);
+      }
+
+      let accountId = PublicKey.fromString(
+        wallet._signingKey().compressedPublicKey.replace("0x", "")
+      ).toAccountId(0, 0);
+      const transferTransaction = new TransferTransaction()
+        .addHbarTransfer(accountId, new Hbar(balance))
+        .addHbarTransfer(AccountId.fromString("0.0.2"), new Hbar(-balance));
+      const tx = await transferTransaction.execute(this.client);
+      let accountNum = `0.0.${aliasedAccountNumCounter}`;
+      if (startup) {
+        aliasedAccountNumCounter++;
+      } else {
+        await tx.getReceipt(this.client);
+
+        const accountInfo = await new AccountInfoQuery({
+          accountId: AccountId.fromEvmAddress(0, 0, wallet.address),
+        }).execute(this.client);
+        accountNum = accountInfo.accountId.toString();
+      }
+      if (async) {
+        params.push({ accountNum: accountNum, wallet: wallet, balance: balance});
+        continue;
+      }
+      this._logAliasAccount(accountNum, balance, wallet);
+    }
+    if (async) return params;
+    this._logAccountivider();
+  }
+
+  /**
+   * @internal
+   * Generate ED25519 accounts.
+   * @param {boolean} async
+   * @param {number} balance
+   * @param {number} num
+   * @param {boolean} startup
+   */
+  async _generateED25519(async, balance, num, startup) {
+    let edAccountNumCounter = 1022;
+    const params = [];
+
+    if (!async) this._logAccountTitle();
+
+    for (let i = 0; i < num; i++) {
+      let wallet = hethers.Wallet.createRandom({ isED25519Type: true });
+      if (startup && this.privateKeysED25519[i]) {
+        wallet = new hethers.Wallet({
+          privateKey: this.privateKeysED25519[i],
+          isED25519Type: true,
+        });
+      }
+      params.push(await this._createAccount(
```

Comment:
similar to comment in `_generateECDSA`

If `async` then `params` should be an array of promises and after the for loop `await Promise.all(params)`